db/schema.rb

```diff
@@ -240,6 +240,7 @@
     t.string   "static_emails_on_automated_deploy_failure",    limit: 255
     t.string   "datadog_monitor_ids",                          limit: 255
     t.string   "jenkins_job_names",                            limit: 255
+    t.integer  "next_stage_id"
```

Comment:
maybe just serialize :next_stage_ids, Array ;)

I like the idea since it would keep stages atomic ...



On Fri, May 29, 2015 at 9:27 AM, Gabe Martin-Dempesy <
notifications@github.com> wrote:

> In db/schema.rb
> <https://github.com/zendesk/samson/pull/425#discussion_r31340310>:
>
> > @@ -240,6 +240,7 @@
> >      t.string   "static_emails_on_automated_deploy_failure",    limit: 255
> >      t.string   "datadog_monitor_ids",                          limit: 255
> >      t.string   "jenkins_job_names",                            limit: 255
> > +    t.integer  "next_stage_id"
>
> How do you feel about reversing the direction of this association from a
> belongs_to to a has_many? That could allow for a tree of host groups,
> e.g. after pod 1, deploy to pods 2, 3, and 5 in parallel.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/425/files#r31340310>.
>